tests/robustness/model/history.go

```diff
@@ -488,36 +486,10 @@ func (h History) Len() int {
 
 func (h History) Operations() []porcupine.Operation {
 	operations := make([]porcupine.Operation, 0, len(h.operations))
-	maxTime := h.lastObservedTime()
```

Comment:
@serathius Because of this removal, the unit tests would have to be updated, due to [this pre-condition](https://github.com/etcd-io/etcd/blob/main/tests/robustness/model/history.go#L312) failing.

But if I am not mistaken, each client is sending the requests sequentially, so the test should be ok to be updated this way, right?

Reply:
Yes, updating unit tests here makes sense.